samples/subsys/usb/hid-cdc/src/main.c

```diff
@@ -0,0 +1,868 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr.h>
+#include <device.h>
+#include <gpio.h>
+#include <uart.h>
+#include <string.h>
+#include <random/rand32.h>
+
+#include <usb/usb_device.h>
+#include <usb/class/usb_hid.h>
+#include <usb/class/usb_cdc.h>
+
+#define LOG_LEVEL LOG_LEVEL_DBG
+LOG_MODULE_REGISTER(main);
+
+#ifdef SW0_GPIO_CONTROLLER
```

Comment:
Why do you redefine all these macros?
If it's to shorten the syntax I am not sure it's worth to add 50 lines of macros.